src/screens/HyperSwitch/Settings/BusinessMapping/BusinessProfile.res

```diff
@@ -111,39 +110,31 @@ let make = (
     updateMerchantDetails(values)->ignore
     Js.Nullable.null
   }
-  let tableHeaderText = isFromWebhooks ? "Webhooks" : "Business Profiles"
 
   <PageLoaderWrapper screenState>
     <UIUtils.RenderIf condition=isFromSettings>
       <div className="relative h-full">
         <div className="flex flex-col-reverse md:flex-col">
           <PageUtils.PageHeading
-            title=tableHeaderText
-            subTitle={isFromWebhooks
-              ? "Set up and monitor transaction webhooks for real-time notifications."
-              : "Add and manage profiles to represent different businesses across countries."}
+            title="Business Profiles"
+            subTitle="Add and manage profiles to represent different businesses across countries."
           />
           <LoadedTable
             title="Business profiles"
             hideTitle=true
             resultsPerPage=7
             visibleColumns
-            entity={businessProfileTabelEntity(isFromWebhooks)}
+            entity={businessProfileTabelEntity()}
```

Comment:
disable the click action in the Business profiles table

Reply:
That's the requirement Jeeva